src/components/Login/Login.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import Layout from "components/Layout/Layout";
+
+export default function Login() {
+  return (
+    <Layout sidebar={false}>
+      <h1>Login</h1>
+      <p>Please log in to view and manage your models.</p>
+      <button className="p-button--positive">Log in</button>
```

Comment:
just fyi, maas-ui has a vanilla `Button` you might find useful.

https://github.com/canonical-web-and-design/maas-ui/tree/master/ui/src/app/base/components/Button